### ISSUE TITLE
Redesign: Neobrutalism

### ISSUE THREAD

**Issue by petersopko** [kodadot/nft-gallery#3950]
It seems to me that redesign info / issues are scattered on too many places with too much text, let's have one simple issue for tracking what's currently happening. I'll keep this updated.

Related discussion:
- [ ] #3710

Landing:

- [x] #3844
- [x] #3944
- [x] #3945
  - [x] #3947 
  - [x] #3928

Navbar and Footer (should be across whole app, not just landing):
- [x] #3946
- [x] #3948

Old redesign ideas are tracked here:
- [ ] #3956

**Comment by prachi00** [comment]
I see that the hover states for urls is missing in the design @petersopko @yangwao

**Comment by yangwao** [comment]
> I see that the hover states for urls is missing in the design @petersopko @yangwao

for design issues best ping @exezbcz

**Comment by exezbcz** [comment]
> I see that the hover states for urls is missing in the design @petersopko @yangwao

Hi, sorry for the late reply, yes, for design it is better to ping me. There is no hover in figma as it comes from a system that repeats. Therefore hover on text will cause bold and socials will change bg color to accent-light. 

Thanks for the ping @yangwao

**Comment by preschian** [comment]
hi @exezbcz do we have a disabled state UI? I think we need that for the arrow in the carousel when reaching the start/end carousel pages, similar to the current implementation in production

**Comment by preschian** [comment]
@exezbcz does the navbar UI final already? If I compare it with the current implementation missing some buttons, such as history, language, dark mode, and chain selection (is it moved to the landing page section?). and how does the search bar in the navbar work, because on the desktop seems missing in the navbar?

**Comment by exezbcz** [comment]
Hey @preschian, good questions, I didn't think specifically about the disabled state for arrows, we can resolve it quickly how it's done on multiple pages, and remove the element once you reach the end of the carousel. As for the search bar in the navbar, it's in Figma in the "components + prototypes" subpage. Maybe I did not make it clear enough, sorry for that. Missing elements in nav - yup, that's correct, we are currently missing language, dark mode, and network. History is built in search bar results, that's the state if you only click on the search bar - see search bar prototype in Figma file. We dropped language bcs we think it will be better in profile settings and just to make nav bar cleaner. The dark mode could be here - didn't put it there, I will think about it. The network button isn't necessary, we are doing landing with NFTs from all networks - as mentioned in #3710.  Thanks!
